nntrainer/layers/layer_node.cpp

```diff
@@ -129,21 +129,73 @@ class SharedFrom : public Name {
  *
  */
 LayerNode::~LayerNode() = default;
+  
+/**
+ * @brief get the compute engine property from property string vector
+ *  : default is CPU
+ * @return LayerComputeEngine Enum : CPU, GPU, QNN
+ *
+ */
+ml::train::LayerComputeEngine
+getComputeEngine(const std::vector<std::string> &props) {
```

Comment:
Function prototype for `getComputeEngine` is missing in header. Isn't it required to be declared in header?